pwintools.py

```diff
@@ -351,36 +346,38 @@ def recv_thread():
 
 class Pipe(object):
     """Windows pipe support"""
-    def __init__(self, bInheritHandle = 1):
+
+    def __init__(self, bInheritHandle=1):
         attr = SECURITY_ATTRIBUTES()
         attr.lpSecurityDescriptor = 0
         attr.bInheritHandle = bInheritHandle
         attr.nLength = ctypes.sizeof(attr)
         self._rpipe, self._wpipe = CreatePipe(attr)
-        self.timeout = 500 # ms
-        self.tick = 40 # ms
-        
-    def get_handle(self, mode = 'r'):
+        self.timeout = 500  # ms
+        self.tick = 40  # ms
+
+    def get_handle(self, mode='r'):
         """get_handle(mode = 'r') returns the 'r'ead / 'w'rite HANDLE of the pipe"""
         if mode and mode[0] == 'w':
             return self._wpipe
         return self._rpipe
-        
+
     def __del__(self):
-        windows.winproxy.CloseHandle(self._rpipe)

```

Comment:
Why do you comment this code ?

Reply:
For debug purpose. I need a code that simply work before make it completely work.

Reply:
I have fixed it. get a look.